build/Build.cs

```diff
@@ -106,8 +106,7 @@ class Build : NukeBuild
                 .SetOutputDirectory(ArtifactsDirectory)
                 .EnableNoBuild()
                 .DisableIncludeSymbols()
-                .SetVerbosity(DotNetVerbosity.Normal)
-                .SetProperty("NuspecProperties", $"Version={OctoVersionInfo.NuGetVersion}"));
```

Comment:
the client package didnt have a nuspec file so this wasnt actually doing anything anyway